index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Green Earth Summit</title>
+</head>
+<body>
+    <navbar>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
+        <div class="hamburger">
+            <ul>
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+        </div>
+        <div  class="flex" >
+            <a  class="logo" href="index.html"><img  id="logo"  src="assets/Earth.png"/></a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+              <a class="nav-link" id="nav1"  href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+            <li class="nav-item">
+              <a class="nav-link"  href="#">News</a>
+            </li>         
+            <li class="nav-item GE">
+                <a class="nav-link" id="ge" href="#">Campaign</a>
+              </li>    
+          </ul>
+          </div>
+    </navbar>
+   <!--Header-->
+   <header>
+    <h1 class="heading1">"Hello!Sharing world"</h1>
+    <h1 class="heading2">Green Earth Global Summit 2023</h1>
+    <p  class="parag">Welcome to our global annual summit that will be held virtually for ease of accessibility at your comfort.
+        For those who want to attend in person, it will be held at Uhuru Park, Nairobi,Kenya. </p>
+    <!-- Today's date -->
```

Comment:
- [ ] In this [review link](https://github.com/fatmahussein/Green-Earth-Summit/pull/4#discussion_r1251675983) the previous reviewer asked that you implement the text to show as provided by the project design but you have not yet implemented this change. I have added an example css code below which you can use to implement this design 
- *example css code 
- The example code below requires you to use an image in the url() 
```css
.animate-text {
   background-image: linear-gradient(rgba(255, 38, 0, 0.76), rgba(255, 51, 0, 0.575)), url(../images/it.jpeg); 
  background-repeat: repeat;
  background-clip: border-box;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  -webkit-font-smoothing: antialiased;
 }
```